dotCMS/src/main/java/com/dotcms/util/pagination/CategoriesPaginator.java

```diff
@@ -0,0 +1,59 @@
+package com.dotcms.util.pagination;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+import com.dotcms.repackage.com.google.common.annotations.VisibleForTesting;
+import com.dotcms.repackage.org.apache.commons.lang.StringUtils;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.RoleAPI;
+import com.dotmarketing.business.UserAPI;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.categories.business.CategoryAPI;
+import com.dotmarketing.portlets.categories.business.PaginatedCategories;
+import com.dotmarketing.portlets.categories.model.Category;
+import com.liferay.portal.model.User;
+
+/**
+ * Created by freddyrodriguez on 8/16/17.
+ */
+public class CategoriesPaginator implements Paginator<Category> {
+
+    private CategoryAPI categoryAPI;
+    private int lastTotalRecords;
+
+    @VisibleForTesting
+    public CategoriesPaginator(CategoryAPI categoryAPI){
+        this.categoryAPI = categoryAPI;
+    }
+
+    public CategoriesPaginator(){
+        this(APILocator.getCategoryAPI());
+    }
+
+    @Override
+    public long getTotalRecords(String filter) {
+        return lastTotalRecords;
+    }
+
+    @Override
+    public Collection<Category> getItems(User user, String filter, int limit, int offset, String orderby, OrderDirection direction, Map<String, Object> extraParams) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 166 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)